washingtonpost/ArcAds
# Add Support for Collapsing Empty Divs

## On this branch
- Added support for collapsing empty divs.

## To test
1. Pull in this new ads branch into your repo
2. Confirm ads still render
3. Ads ability to pass `dfp.collapseEmptyDivs` to the ArcAds constructor
